tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -37,10 +39,27 @@ func getLatestCirrosImageURL() (string, error) {
 	return imageURL, nil
 }
 
+func vmPoweredOff(vmnamespace, vmname string) VerificationFunction {
+	return VerificationFunction(func(ocClient client.Client, namespace string) error {
+		isOff := func() bool {
+			status, err := lib.GetVmStatus(dynamicClientForSuiteRun, vmnamespace, vmname)
+			if err != nil {
+				log.Printf("Error getting VM status: %v", err)
```

Comment:
should there be a return on error?

Reply:
I think this could include an error, but I don't know that it would meaningfully improve the Eventually. I guess there might be cases where it would quit faster than the timeout, but then it would also need to check for errors that it should be expected to ride through. I will see if I can think of something.

Reply:
I just see some possible logic problem. Saying first that there is err getting status and then without return logging off vm status and returning "Stopped", for me if there was error getting status the return of the status should not be "Stopped" but something unknown